src/common/input/input_event.cpp

```diff
@@ -43,8 +45,8 @@ std::string input_event_type_to_string(MirInputEventType input_event_type)
             return "mir_input_event_type_touch";
         case mir_input_event_type_pointer:
             return "mir_input_event_type_pointer";
-        default:
-            abort();
+    default:
+            std::unreachable();
```

Comment:
Also here. Make it a compile error on clang?

Also, inconsistent indentation (although what you've added is more correct according to the style guide)